data-pipeline/src/trace_exporter.rs

```diff
@@ -254,7 +253,9 @@ impl TraceExporter {
         self.check_agent_info();
         match self.input_format {
             TraceExporterInputFormat::Proxy => self.send_proxy(data, trace_count),
-            TraceExporterInputFormat::V04 => self.send_deser_ser(data),
+            TraceExporterInputFormat::V04 => {
+                self.send_deser_ser(tinybytes::Bytes::copy_from_slice(data))
+            } //todo do we want to copy?
```

Comment:
```suggestion
// TODO: APMSP-1582 - Refactor data-pipeline-ffi so we can leverage a type that implements tinybytes::UnderlyingBytes trait to avoid copying                self.send_deser_ser(tinybytes::Bytes::copy_from_slice(data))
            } 
```